### DISCUSSION TITLE
Problem to the test render form with useFieldArray using jest

### DISCUSSION THREAD

**Discussion by julio3266** [react-hook-form/react-hook-form#7828] [Show and tell]
I'm trying to do a test using jest, but I received the following error: 
error: encountered a declaration exception
<img width="652" alt="Captura de Tela 2022-02-15 às 07 43 52" src="https://user-images.githubusercontent.com/52897217/154046141-89b62203-d0b3-4c2a-a502-bf339be39263.png">

**Comment by Moshyfawn** [comment]
We can't know what's wrong with your code without a reproduction.
Please, see how the tests are done [in RHF repo](https://github.com/react-hook-form/react-hook-form/tree/master/src/__tests__/useFieldArray).
Also, see #2649 discussion

**Reply by julio3266** [reply]
@Moshyfawn the link of discussion is wrong rsrs

**Reply by Moshyfawn** [reply]
My bad, fixed it

**Reply by julio3266** [reply]
@Moshyfawn  i need create a mock for useFieldArray?
